templates/_helpers.tpl

```diff
@@ -46,54 +46,57 @@ app.kubernetes.io/managed-by: {{ .Release.Service }}
 {{/*
 Selector labels
 */}}
-{{- define "fcrepo.selectorLabels" -}}
+{{- define "fcrepo.selectorLabels" }}
 app.kubernetes.io/name: {{ include "fcrepo.name" . }}
 app.kubernetes.io/instance: {{ .Release.Name }}
 {{- end }}
 
 {{/*
 Create the name of the service account to use
 */}}
-{{- define "fcrepo.serviceAccountName" -}}
+{{- define "fcrepo.serviceAccountName" }}
 {{- if .Values.serviceAccount.create }}
 {{- default (include "fcrepo.fullname" .) .Values.serviceAccount.name }}
 {{- else }}
 {{- default "default" .Values.serviceAccount.name }}
 {{- end }}
 {{- end }}
 
-{{- define "fcrepo.postgresql.fullname" -}}
-{{- $name := default "postgresql" .Values.postgresql.nameOverride -}}
-{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
-{{- end -}}
+{{- define "fcrepo.postgresql.fullname" }}
+{{- $name := default "postgresql" .Values.postgresql.nameOverride }}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" }}
+{{- end }}
 
 {{/*
 Return PostgreSQL host
 */}}
-{{- define "fcrepo.postgresql.host" -}}
-{{- include "fcrepo.postgresql.fullname" . }}
-{{- end -}}
+{{- define "fcrepo.postgresql.host" }}
+{{- if .Values.postgresql.postgresqlHost }}
```

Comment:
maybe always use the chart deployed postgres if you have deployed one? something like:
```suggestion
{{- if not .Values.postgresql.enabled }}
```